packages/riverpod/lib/src/devtool.dart

```diff
@@ -1,7 +1,50 @@
-// ignore_for_file: public_member_api_docs
+// ignore_for_file: public_member_api_docs, invalid_use_of_visible_for_testing_member
```

Comment:
What is this change for?

Reply:
this is related to the other question that you made https://github.com/rrousselGit/riverpod/pull/1471#discussion_r913596932 . I can totally remove it if I make it a `part` of [provider_base.dart](https://github.com/rrousselGit/riverpod/pull/1471/files/5ccdcfd9142e7254039d697113dfbe92caa7f7c9#diff-7c816cec9b846943e23c9577fcb45a6bdeed95a20b7a87f333f24f28e8201b7a).

Reply:
removed